Buildings/Controls/OBC/Utilities/PIDWithAutotuning/AutoTuner/AMIGO/package.mo

```diff
@@ -0,0 +1,6 @@
+within Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner;
+package Amigo "Package with components related to AMIGO tuner"
+annotation (Documentation(info="<html>
+<p>This package contains the blocks to implement the AMIGO tuner. </p>
```

Comment:
Better to add more detailed introduction of how the "Amigo" tuner work. Please also add the reference link.

Reply:
Added high-level information